helm/presets/aws.yaml

```diff
@@ -81,13 +99,16 @@ presto:
       user: ""
       password: ""
 
+# Cert Manager subchart
 cert_manager:
   enabled: true
 
-global:
-  objectStore:
-    region: "us-east-2"
-    accessKey: ""
-    accessSecret: ""
```

Comment:
If we're removing all references to these, then they should be validated via the `required` macro. That way it can't be a surprise.

Reply:
Also, you're breaking an implicit contract since we wouldn't be defaulting to us-east-2. Maybe that's fine, especially this early, but it's worth noting.

The documentation (README, etc) probably needs updated to reflect these changes.

Reply:
This is still in the file; just moved up above the subchart values references.

Reply:
Ah, damn diff got me. Thanks.